pyproject.toml

```diff
@@ -24,7 +24,7 @@ chmigrate = 'clickhouse_migrate.runner:run'
 
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.8"
```

Comment:
You need to change the version of python only in this file. And do not touch it in actions.